lib/themes/themes.js

```diff
@@ -0,0 +1,42 @@
+import themeDarkSvg from 'design-system/assets/img/pm-images/theme-dark.svg';
+import themeLightSvg from 'design-system/assets/img/pm-images/theme-light.svg';
+import lightTheme from 'design-system/_sass/pm-styles/_pm-light-theme.scss';
+import themeBlueSvg from 'design-system/assets/img/pm-images/theme-blue.svg';
+import blueTheme from 'design-system/_sass/pm-styles/_pm-blue-theme.scss';
+import themeTestSvg from 'design-system/assets/img/pm-images/theme-test.svg';
+
+export const PROTON_THEMES = {
+    DARK: {
+        label: 'Dark (Default)',
+        identifier: '/* dark-theme */',
+        src: themeDarkSvg,
+        theme: '',
+        customizable: false
+    },
+    LIGHT: {
+        label: 'Light',
+        identifier: '/* light-theme */',
+        src: themeLightSvg,
+        theme: lightTheme,
```

Comment:
is it possible to lazy load the CSS ?

Reply:
Dynamically importing the scss files seems to give some trouble with sass variables. Also it may be better to load all the themes at the beginning so that the theme change happens faster. The themes are not heavy